scripts/review/verifyDeployment.ts

```diff
@@ -84,6 +84,7 @@ async function main() {
 
       debug(`• ${deployment} deployment OK`);
     }
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
> 87

From Node.js 15 and up you can use the [Timers Promises API](https://nodejs.org/api/timers.html#timers-promises-api). You don't have to promisify setTimeout or rely on a 3rd party library anymore.

import { setTimeout } from 'node:timers/promises';

await setTimeout(1000);

source:
https://stackoverflow.com/a/71442573/7820085

Reply:
with this you can write in a way where it reads much better, example:
```
import { setTimeout as sleep } from 'node:timers/promises';

await sleep(1000)
```

Reply:
Updated 👍🏾